### ISSUE TITLE
Logged out users can see archived forum posts

### ISSUE THREAD

**Issue by Barsik4** [Avicus/Issues#1376]
https://gyazo.com/fb1bd52430980a77df06074f292f7d66

**Comment by smitdalt** [comment]
Can you see this while logged in?

**Comment by Barsik4** [comment]
No, I can not see archived forum posts whilst being logged in. [![Screenshot from Gyazo](https://gyazo.com/9f928a3c565eb7b69d4417f40fb2db76/raw)](https://gyazo.com/9f928a3c565eb7b69d4417f40fb2db76)
